packages/backend/src/handlers/remote-schemas/resolvers/getBoxProfile/resolver.ts

```diff
@@ -47,3 +50,13 @@ function getImage(
   }
   return '';
 }
+
+function getCollectiblesFavourites(
+  collectiblesFavorites: Array<CollectiblesFavorite> | null | undefined,
```

Comment:
the schema is
`collectiblesFavorites: [CollectiblesFavorites!]`

so maybe the return type should be only Array<CollectiblesFavorite> and always return an empty array if theres nothing in the profile

Reply:
so do you want me to change the schema to `collectiblesFavorites: [CollectiblesFavorites!]!`

Reply:
I wanted the schema to remain uniform with the rest of the values:
```
  type BoxProfile {
    ethereumAddress: String
    name: String
    description: String
    location: String
    job: String
    emoji: String
    imageUrl: String
    coverImageUrl: String
    website: String
    collectiblesFavorites: [CollectiblesFavorites!]
  }
```

Reply:
also note the `type CollectiblesFavorite` in this line is different from our typeDef, and comes from 3box.

Reply:
Ok my bad, didn't see that all the properties were nullable in the GraphQL schema. It's ok like this then !